internal/cephfs/volume.go

```diff
@@ -119,9 +119,17 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 	return &subvol, nil
 }
 
+type ResizeStatus int64
+
+const (
+	resizeUnknown ResizeStatus = iota
+	resizeSupported
+	resizeUnsupported
+)
+
 type localClusterState struct {
 	// set true if cluster supports resize functionality.
-	resizeSupported *bool
+	resizeSupported ResizeStatus
```

Comment:
probably better to rename this attribute, and call it `resizeState` so that it does not conflict with the naming of the const.

Reply:
okay